packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -318,14 +360,19 @@ class _CupertinoCheckboxState extends State<CupertinoCheckbox> with TickerProvid
   @override
   Widget build(BuildContext context) {
     // Colors need to be resolved in selected and non selected states separately.
+    // The `states` getter constructs a new set every time, which is therefore safe to edit in place.
     final Set<WidgetState> activeStates = states..add(WidgetState.selected);
     final Set<WidgetState> inactiveStates = states..remove(WidgetState.selected);
 
     // Since the states getter always makes a new set, make a copy to use
     // throughout the lifecycle of this build method.
     final Set<WidgetState> currentStates = states;
 
-    final Color effectiveActiveColor = _defaultFillColor.resolve(activeStates);
+    final Color effectiveActiveColor = widget.fillColor?.resolve(activeStates)
+            ?? _defaultFillColor.resolve(activeStates);
```

Comment:
The indentation here feels weird, does this follow the style guide?

Reply:
I'm not sure if the style guide has an enforcing rule but usually we use 2 spaces or 4 spaces in this kind of cases.
```suggestion
        ?? _defaultFillColor.resolve(activeStates);
```

Reply:
True. All the indentation stipulations in the style guide seem to suggest two spaces.

Reply:
Not really. For example, https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#prefer-a-maximum-line-length-of-80-characters provided an example of 
```
final int c =
    a.very.very....
```

Although it's in the  BAD example section, it still shows that _if_ we want a new line, the new line _can_ use 4 spaces.

Reply:
Also in https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#align-expressions there is an example that shows what happens if we want to indent at the middle of an expression:
```
// BAD:
return foo.x == x &&
    foo.y == y &&
    foo.z == z;
```